src/sentry/spans/grouping/strategy/base.py

```diff
@@ -84,9 +84,11 @@ def handle_default_fingerprint(self, span: Span) -> Sequence[str]:
         return span_group
 
 
-def span_op(op_name: str) -> Callable[[CallableStrategy], CallableStrategy]:
+def span_op(op_name: Union[str, Sequence[str]]) -> Callable[[CallableStrategy], CallableStrategy]:
```

Comment:
This is more of a nit and doesn't really matter at this scale but using a set instead of a sequence might make more sense here.

Reply:
Fair enough! I'll leave it as-is for now, then, but noted!